bolt/lib/Core/ParallelUtilities.cpp

```diff
@@ -102,12 +102,15 @@ inline unsigned estimateTotalCost(const BinaryContext &BC,
 
 } // namespace
 
-ThreadPoolInterface &getThreadPool() {
+ThreadPoolInterface &getThreadPool(const unsigned int ThreadsCount) {
```

Comment:
just unsigned is fine.
For example:
heavyweight_hardware_concurrency(unsigned ThreadCount = 0)